api/lib/resolvers.js

```diff
@@ -294,6 +300,41 @@ const transactionMetadata = async (
   }
 }
 
+const storeUser = async (_, { idToken, premium }) => {
+  try {
+    const decodedToken = await firebaseAdmin.auth().verifyIdToken(idToken)
+    // get user creation date
+    const userRecord = await firebaseAdmin.auth().getUser(decodedToken.uid)
+    // check if user already exists in DB
+    const storedUser = await database(config)('').getUser(decodedToken.uid)
+    // check if user already has premium as a custom claim
+    if (!userRecord.customClaims) {
+      // set premium field
+      await firebaseAdmin
+        .auth()
+        .setCustomUserClaims(decodedToken.uid, { premium: premium || false })
+    }
+    const user = {
+      uid: decodedToken.uid,
+      email: decodedToken.email,
+      premium: premium || false,
+      createdAt: userRecord.metadata.creationTime,
```

Comment:
as we have `createdAt` and `lastActive` in firebase, let's not store them in our db

Reply:
it was in the issue description 🤷